### PR TITLE
en: simplify kind tutorial

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Simplify the doc for deploying tidb using kind.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/263
- Other reference link(s):

### CODE REVIEW

File: en/deploy-tidb-from-kubernetes-kind.md [hunk 1]
```diff
@@ -19,143 +19,77 @@ kind is a tool for running local Kubernetes clusters using Docker containers as
...
 {{< copyable "shell-regular" >}}
 
 ```shell
-helm search pingcap -l
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.0-rc.3/manifests/crd.yaml && \
+kubectl get crd tidbclusters.pingcap.com
 ```
 
-TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
+The following is an example of installing TiDB Operator using Helm 3:
```
Comment (TomShawn) [inline en/deploy-tidb-from-kubernetes-kind.md:67]:
```suggestion
Install TiDB Operator. The following is an example of installing TiDB Operator using Helm 3:
```

File: en/deploy-tidb-from-kubernetes-kind.md [hunk 2]
```diff
@@ -19,143 +19,77 @@ kind is a tool for running local Kubernetes clusters using Docker containers as
...
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-helm repo add pingcap https://charts.pingcap.org/
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
 ```
 
-Use `helm search` to search the chart provided by PingCAP:
+TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create CRDs, such as `TidbCluster`:
```
Comment (TomShawn) [inline en/deploy-tidb-from-kubernetes-kind.md:58]:
```suggestion
Create [CRDs (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). TiDB Operator uses CRDs to extend Kubernetes. Therefore, to use TiDB Operator, you must first create CRDs, such as `TidbCluster`:
```

File: en/deploy-tidb-from-kubernetes-kind.md [hunk 3]
```diff
@@ -19,143 +19,77 @@ kind is a tool for running local Kubernetes clusters using Docker containers as
...
-    standard (default)   kubernetes.io/host-path        8m29s
-    ```
+```
+kubectl cluster-info
+```
 
 ## Step 2: Deploy TiDB Operator
 
-Add the official chart repository provided by PingCAP:
+First, install Helm. You can either refer to [Installing Helm](https://helm.sh/docs/intro/install/) or install the latest stable version of Helm 3:
```
Comment (TomShawn) [inline en/deploy-tidb-from-kubernetes-kind.md:50]: Would it be better to use an ordered list?

Comment (ran-huang) [reply en/deploy-tidb-from-kubernetes-kind.md]: Updated.

**Comment by ran-huang** [comment]
@TomShawn PTAL again. Thanks~

**Pull request merged**